centreon/tests/e2e/features/Dashboards/01-dashboard-creation/index.ts

```diff
@@ -195,3 +195,38 @@ Then('the form fields are empty', () => {
   cy.getByLabel({ label: 'Name', tag: 'input' }).should('be.empty');
   cy.getByLabel({ label: 'Description', tag: 'textarea' }).should('be.empty');
 });
+
+Given(
+  "a dashboard with existing widgets in the dashboard administrator user's dashboard library",
+  () => {
+    cy.insertDashboardWithPatch(dashboards.fromDashboardCreatorUser,patchBody);
+    cy.visit(`${Cypress.config().baseUrl}/centreon/home/dashboards`);
+  }
+);
+
+
+When('the dashboard administrator user starts to edit the dashboard', () => {
+      cy.getByLabel({ label: 'view', tag: 'button' }).click();
+});
+
+Then("creates a new dashboard on the previous dashboard's edition page", () => {
+      cy.getByTestId({ testId: 'ArrowDropDownIcon' }).click();
+      cy.getByTestId({ testId: 'AddIcon' }).click();
+      cy.getByLabel({ label: 'Name', tag: 'input' }).type(dashboards.default.name);
+      cy.getByLabel({ label: 'Description', tag: 'textarea' }).type(dashboards.default.description);
+      cy.getByLabel({ label: 'Create', tag: 'button' }).click()
+      cy.wait('@createDashboard')
+
```

Comment:
linting seems wrong : indentation, carriage return..

Reply:
you should probably install eslint on your IDE